### ISSUE TITLE
From Error log: bad argument #2 to '__random' (interval is empty)

### ISSUE THREAD

**Issue by geramy92** [TerraTex-Community/TerraTex-Reallife-Reloaded#595]
- Database Id: 42
- Message: bad argument #2 to '__random' (interval is empty)
- Stacktrace: 
```
stack traceback:
	...atex_reallife\SYSTEM\logging\logging_server_func.lua:16: in function <...atex_reallife\SYSTEM\logging\logging_server_func.lua:15>
```
- Level: ERROR
- Source: USEFULL_SCRIPTS\number_math_functions.lua:65
- LogType: server
- Appeared at User: server

**Comment by geramy92** [comment]
Can only solved after #599 - will automatically open new issue
